test/e2e/specs/specs-wpcom/wp-editor__revisions.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @group gutenberg
+ */
+
+import {
+	DataHelper,
+	BrowserHelper,
+	TestAccount,
+	GutenbergEditorPage,
+	EditorSettingsSidebarComponent,
+	RevisionsComponent,
+	setupHooks,
+	ParagraphBlock,
+} from '@automattic/calypso-e2e';
+import { Page } from 'playwright';
+
+describe( DataHelper.createSuiteTitle( `Editor: Revisions` ), function () {
+	const accountName = BrowserHelper.targetGutenbergEdge()
+		? 'gutenbergSimpleSiteEdgeUser'
+		: 'simpleSitePersonalPlanUser';
+	let gutenbergEditorPage: GutenbergEditorPage;
+	let editorSettingsSidebarComponent: EditorSettingsSidebarComponent;
+	let page: Page;
+
+	setupHooks( async ( args ) => {
+		page = args.page;
+		gutenbergEditorPage = new GutenbergEditorPage( page );
+
+		const testAccount = new TestAccount( accountName );
+		await testAccount.authenticate( page );
+	} );
+
+	it( 'Go to the new post page', async function () {
+		await gutenbergEditorPage.visit( 'post' );
+	} );
+
+	it( 'Enter post title', async function () {
+		gutenbergEditorPage = new GutenbergEditorPage( page );
+		await gutenbergEditorPage.enterTitle( DataHelper.getRandomPhrase() );
+		await gutenbergEditorPage.saveDraft();
+	} );
+
+	it.each( [ { revision: 1 }, { revision: 2 } ] )(
+		'Create revision $revision',
+		async function ( { revision } ) {
+			const blockHandle = await gutenbergEditorPage.addBlock(
+				ParagraphBlock.blockName,
+				ParagraphBlock.blockEditorSelector
+			);
+			const paragraphBlock = new ParagraphBlock( blockHandle );
+			await paragraphBlock.enterParagraph( `Revision ${ revision }` );
+			await gutenbergEditorPage.saveDraft();
+		}
+	);
+
+	it( 'View revisions', async function () {
+		const frame = await gutenbergEditorPage.getEditorFrame();
+		await gutenbergEditorPage.openSettings();
+		editorSettingsSidebarComponent = new EditorSettingsSidebarComponent( frame, page );
+		await editorSettingsSidebarComponent.clickTab( 'Post' );
+		await editorSettingsSidebarComponent.showRevisions();
```

Comment:
Can we add a case here where select the revisions in the sidebar of the revisions modal and verify that the text changes in the modal preview before clicking load? That seems like a low-cost, low-risk, but valuable addition to this spec!

Right now, if that display pane in the modal busted, we wouldn't catch it with this current test!

Reply:
Added step to check contents of the revision diff viewer in 34af05384435710ef0a1e8e95a2b4916646084da.